Tasks/JenkinsQueueJobV3/jenkinsqueuejobtask.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import tl = require('azure-pipelines-task-lib/task');
+import path = require('path');
+import util = require('./util');
+
+import { JobQueue } from './jobqueue';
+
+export class TaskOptions {
+    serverEndpoint: string;
+    serverEndpointUrl: string;
+
+    serverEndpointAuth: tl.EndpointAuthorization;
+    username: string;
+    password: string;
+
+    jobName: string;
+    isMultibranchPipelineJob: boolean;
+    multibranchPipelineBranch: string;
+
+    captureConsole: boolean;
+    // capturePipeline is only possible if captureConsole mode is enabled
+    capturePipeline: boolean;
+
+    pollIntervalMillis: number;
+
+    parameterizedJob: boolean;
+    // jobParameters are only possible if parameterizedJob is enabled
+    jobParameters: string[];
+
+    jobQueueUrl: string;
+    teamJobQueueUrl: string;
+    teamPluginUrl: string;
+
+    teamBuildPluginAvailable: boolean;
+    saveResultsTo: string;
+
+    strictSSL: boolean;
+
+    NO_CRUMB: string;
+    crumb: string;
+
+    shouldFail: boolean;
+    failureMsg: string;
+
+    constructor() {
+        this.serverEndpoint = tl.getInput('serverEndpoint', true);
+        this.serverEndpointUrl = tl.getEndpointUrl(this.serverEndpoint, false);
+        tl.debug('serverEndpointUrl=' + this.serverEndpointUrl);
+        this.serverEndpointAuth = tl.getEndpointAuthorization(this.serverEndpoint, false);
+        this.username = this.serverEndpointAuth['parameters']['username'];
+        this.password = this.serverEndpointAuth['parameters']['password'];
+
+        this.jobName = tl.getInput('jobName', true);
+        this.isMultibranchPipelineJob = tl.getBoolInput('isMultibranchJob', false);
+        if (this.isMultibranchPipelineJob) {
+            this.multibranchPipelineBranch = tl.getInput('multibranchPipelineBranch', true);
+        }
+
+        this.captureConsole = tl.getBoolInput('captureConsole', true);
+        // capturePipeline is only possible if captureConsole mode is enabled
+        this.capturePipeline = this.captureConsole ? tl.getBoolInput('capturePipeline', true) : false;
```

Comment:
I think this is simpler: `this.captureConsole && tl.getBoolInput('capturePipeline', true)`